ESP_ATMod/command.cpp

```diff
@@ -1689,68 +1741,106 @@ void cmd_AT_CIPSSLCERT()
 			Serial.printf_P(MSG_OK);
 			Serial.print('>');
 		}
-		else  // OOM
+		else // OOM
 		{
 			Serial.printf_P(MSG_ERROR);
 		}
 	}
-	else if (inputBuffer[13] == '?' && inputBufferCnt == 16)
+	// Print all certificates
+	else if (inputBuffer[offset] == '?' && inputBufferCnt == offset + 3)
 	{
-		if (CAcert == nullptr)
+		if (CAcert.getCount() == 0)
 		{
-			Serial.println(F("+CIPSSLCERT:no cert"));
+			Serial.println(F("+CIPSSLCERT:no certs loaded"));
+			Serial.printf_P(MSG_ERROR);

```

Comment:
Sorry, the correct place for the OK message is here.

Reply:
Changed this one.